API.md

```diff
@@ -92,6 +92,7 @@ Returns a `Promise` that resolves to [a database instance](#store-api). `name` (
 - `directory` (string): The directory where data will be stored (Default: uses directory option passed to OrbitDB constructor or `./orbitdb` if none was provided).
 - `write` (array): An array of hex encoded public keys which are used to set write acces to the database. `["*"]` can be passed in to give write access to everyone. See the [GETTING STARTED](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md) guide for more info. (Default: uses the OrbitDB instance key `orbitdb.key`, which would give write access only to yourself)
 - `overwrite` (boolean): Overwrite an existing database (Default: `false`)
+- `replicate` (boolean): Subscribe & exchange heads over IPFS PubSub. (Default: `true`)
```

Comment:
I think a better description for this is *"Replicate the database with peers."* as that's what it does atm (see https://github.com/orbitdb/orbit-db/blob/master/src/OrbitDB.js#L146). If set to `false`, OrbitDB won't subscribe to the pubsub topic and thus will not receive any updates from peers nor send updates to peers.

Reply:
How about "Replicate the database with peers over IPFS PubSub"?

Reply:
Sounds good 👍

Reply:
(although technically the *replication* doesn't happen over pubsub 😉but it's fine to describe it as such)

Reply:
ah right - just the exchange of heads. I'll switch to "Replicate the database with peers (requires IPFS PubSub)"?

Just want to inform those who (strangely) are not using PubSub.

Reply:
👍